shared/js/initApp.js

```diff
@@ -1,9 +1,21 @@
+// TODO (brent) - way too many globals
+// TODO (brent) - I wonder if we should sub-namespace dashboard
+/* global script_path, Dialog, CDOSounds, dashboard, appOptions, $, trackEvent, Blockly, Applab, sendReport, cancelReport, lastServerResponse, showVideoDialog, ga*/
```

Comment:
I think the solution here is to also send more dashboard stuff through browserify.